### ISSUE TITLE
Bug in estimators/SEMEstimator.py

### ISSUE THREAD

**Issue by prakharverma** [pgmpy/pgmpy#1803]
### Subject of the issue

Potential bug in the SEM script

### Bugs

* Following line has same variable for `zeta` and `B`, key typo: 
```
https://github.com/pgmpy/pgmpy/blob/2c10319adc66327ec57d0f247b6a07a10be39a23/pgmpy/estimators/SEMEstimator.py#L371
```

* In the following line, `S` is a Tensor so an error is thrown. Should be `sample_cov` or `S.detach().numpy()`
```
https://github.com/pgmpy/pgmpy/blob/2c10319adc66327ec57d0f247b6a07a10be39a23/pgmpy/estimators/SEMEstimator.py#L342
```

**Comment by ankurankan** [comment]
@prakharverma Thanks a lot for reporting the bugs. If you'd like, you're welcome to create a PR fixing these :)

**Comment by jihyeseo** [comment]
Could you assign this to me? Thanks!

**Comment by jihyeseo** [comment]
I fixed the typo and data type in https://github.com/pgmpy/pgmpy/pull/1901, but I did not turn on the skipped test cases, since the values of B and zeta seem incorrect in test cases. I opened another issue. https://github.com/pgmpy/pgmpy/issues/1902
